src/foam/u2/wizard/controllers/IncrementalWizardController.js

```diff
@@ -72,8 +72,22 @@ foam.CLASS({
 
         for ( let action of wizardletActions ) {
           if ( action.name === 'goNext' ) {
-            goNextAction = this.GO_NEXT.clone().copyFrom(action);
+
+            // If the class matches we can copy the original NEXT action
+            if ( action.cls_ == foam.core.Action ) {
```

Comment:
How about foam.core.Action.isInstance(action) so that it also works with subclasses?

Reply:
Not matching subclasses is intentional - I only want this to do the `.clone().copyFrom` approach with an exact match. (otherwise `code` of a subclass ends up on an `Action` and it breaks)

Reply:
specifically `AlternateFlowAction` - it's broken without this change